packages/cardano-services/README.md

```diff
@@ -144,12 +144,12 @@ RABBITMQ_SRV_SERVICE_NAME=some-domain-for-rabbitmq \
 ### Generating Projector Schema Migrations
 
 1. Start local projection dependencies (e.g. `yarn preprod:up cardano-node-ogmios postgres`).
-1. Initialize base schema by running `start-projector` with **all** projections enabled. *Hint: you can use `git` to checkout a previous commit if you already updated entities and skipped this step*
-2. Create new entities (or update existing ones).
-3. Include any new entities in `entities` object at `src/Projection/prepareTypeormProjection.ts`.
-4. Run `yarn generate-migration`
-5. Inspect the generated migration and add a static `entity` property. *Hint: you may need to split generated migration into multiple migrations manually if it affects more than 1 entity*
-6. Export the new migration(s) from `migrations` array at `src/Projections/migrations/index.ts`
+2. Initialize base schema by running `start-projector` with **all** projections enabled. _Hint: you can use `git` to checkout a previous commit if you already updated entities and skipped this step_
```

Comment:
:smile:

Reply:
Sorry, @rhyslbw , I didn't understood what sparked your reaction.

Was it the repeated point 1 or something introduced by my change?

Thank you

Reply:
Sorry, that was unnecessary. I was smiling at the docs "bug" you found. Nice job

Reply:
As I understood, no actions are required here